eng/targets/Settings.props

```diff
@@ -50,6 +50,13 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <!-- Nuget Settings -->  
+  <PropertyGroup>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile> <!-- Enable deterministic restore -->
+    <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation> <!-- Enable faster restore times -->
```

Comment:
Should we remove this script argument now?

https://github.com/dotnet/roslyn/blob/c76abe412b4901619fd411fd94f06f3667c087af/eng/build.ps1#L265

Reply:
does the toolset restore use whatever is in our props file or is it run before that?